java-checks/src/test/files/checks/DoubleCheckedLockingAssignment.java

```diff
@@ -0,0 +1,123 @@
+package test;
+
+public class DoubleCheckedLockingAssignment {
+
+  private volatile List<String> strings;
```

Comment:
I know we don't really care about the type of the symbol itself, but could you import `java.util.List` and `java.util.ArrayList` to be sure that unresolved types does not play a role?